### PR TITLE
🚀: Add ERD web project to Vercel deployment workflow

### PR DESCRIPTION
**Pull request by sasamuku**
## Summary

Add deploy workflow for Liam ERD Web.

### CODE REVIEW

**Comment by MH4GF** [comment]
@sasamuku I would like to know if there is a URL I can try!
(I don't have a public SQL file URL 😇)

**Comment by sasamuku** [comment]
Here is sample!
```
raw.githubusercontent.com/ekylibre/ekylibre/refs/heads/main/db/structure.sql
```

File: frontend/apps/erd-web/next.config.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import type { NextConfig } from 'next'
 
 const nextConfig: NextConfig = {
   /* config options here */
+  transpilePackages: ['@liam-hq/db-structure', '@liam-hq/erd-core'],
```
Comment (MH4GF) [inline frontend/apps/erd-web/next.config.ts:5]: Is this necessary after all?

**Review by MH4GF — APPROVED** [review]
LGTM, but I wrote one comment!

**Comment by MH4GF** [comment]
@sasamuku Is it correct? but return 404.

https://liam-erd-3gjijn9bu-route-06-core.vercel.app/erd/p/raw.githubusercontent.com/ekylibre/ekylibre/refs/heads/main/db/structure.sql

File: frontend/apps/erd-web/next.config.ts (hunk 1 above)
Comment (sasamuku) [reply frontend/apps/erd-web/next.config.ts]: Not necessary 👍
290ce475503a87acbcb275a66e75b0298a9d3c68

**Comment by sasamuku** [comment]
@MH4GF 
It looks like workflow is success, but actually not deployed for some reason🤔
let me check.

**Comment by sasamuku** [comment]
@MH4GF 
Sorry for confusing, now you can see via preview 🙏
https://liam-erd-am4f1rh9t-route-06-core.vercel.app/erd/p/raw.githubusercontent.com/ekylibre/ekylibre/refs/heads/main/db/structure.sql

**Comment by MH4GF** [comment]
I don't know why the deployments are in error. but it looks good 🤔 

スクリーンショット 2024-12-25 17 37 52

**Review by MH4GF — APPROVED** [review]
Looks great!
If merging doesn't change the problem, let's handle it.

**Comment by sasamuku** [comment]
@MH4GF 
>I don't know why the deployments are in error. but it looks good 🤔

This is why I've changed `matrix.apps.name` from `@liam-hq/erd-web` to `erd-web`.
Never mind👍

**Comment by MH4GF** [comment]
@sasamuku Oh! I understand 😄

**Pull request merged**